Client/InfluxDBClient.cs

```diff
@@ -285,19 +285,21 @@ public class InfluxDBClient : IInfluxDBClient
         /// </summary>
         /// <param name="host">The URL of the InfluxDB server.</param>
         /// <param name="token">The authentication token for accessing the InfluxDB server.</param>
+        /// <param name="authScheme">Token authentications scheme.</param>
         /// <param name="organization">The organization name to be used for operations.</param>
         /// <param name="database">The database to be used for InfluxDB operations.</param>
         /// <example>
         /// using var client = new InfluxDBClient("https://us-east-1-1.aws.cloud2.influxdata.com", "my-token", "my-org", "my-database");
```

Comment:
Please add  parameters name (it will be better for future incompatible changes:

 ```suggestion
        /// using var client = new InfluxDBClient(host: "https://us-east-1-1.aws.cloud2.influxdata.com", token: "my-token", organization: "my-org", database: "my-database");
```